elements/pfe-sass/extends/_typography_extends.scss

```diff
@@ -0,0 +1,55 @@
+/// EXTENDS
+/// @see {mixin} pfe-text 
+
+%pfe-content-base {
+    font-family: pfe-var(font-family);
+    line-height: pfe-var(line-height);
+    font-weight: pfe-var(font-weight--light);
+}
+
+/// CONTENT BLOCK
+%pfe-content--p {
+    @include pfe-text(md); 
+}
+
+%pfe-content--text--spacing-block {
```

Comment:
What is a spacing block?  Can you add a note that describes when this should be used?  Do I remember that the content-spacer being used differs based on the text size?  Maybe that reduces the usability of this extend?